templates/Dataland/v2/download-card.html.twig

```diff
@@ -0,0 +1,18 @@
+{% if dataset.availabilityStatus == constant('App\\Entity\\DatasetSubmission::AVAILABILITY_STATUS_PUBLICLY_AVAILABLE')
```

Comment:
The file manager is still showing up for restricted remotely hosted datasets. I also don't think the file manager should be shown at all for any remotely-hosted datasets.

Reply:
You're right

Reply:
Fixed it